modules/nw-sctp-verifying.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+:image: registry.access.redhat.com/ubi8/ubi
+
+ifdef::openshift-origin[]
+:image: fedora:31
+endif::[]
+
+[id="nw-sctp-verifying_{context}"]
+= Verifying Stream Control Transmission Protocol (SCTP) is enabled
+
+You can verify that SCTP is working on a cluster by creating a Pod with an application that listens for SCTP traffic, associating it with a Service, and then connecting to the exposed service.
+
+.Prerequisites
+
+* Access to the Internet from the cluster to install the `nc` package.
```

Comment:
Is the `nc` package available by default to RHEL? Support's filed several tickets asking that I purge 'jq' commands from the collection because the package isn't always available, so it might be worth checking before we publish this method.

Reply:
@kalexand-rh, I wondered that myself. The UBI image used for the verification pods includes `nc`.